app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+require 'date'
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
+  validates :password, presence: true
+
+  has_many :events, foreign_key: 'creator_id', class_name: 'Event'
```

Comment:
You don't need to specify the class name since it's already `has_many :events`. You only specify the class name when the association name is different from the class name.